h2/connection.py

```diff
@@ -1450,7 +1450,7 @@ def receive_data(self, data):
         :returns: A list of events that the remote peer triggered by sending
             this data.
         """
-        self.config.logger.debug(
+        self.config.logger.trace(
```

Comment:
@Kriechi I don't follow this change, trace doesn't seem to do anything different - so why change it?

Reply:
When using a logger that prints "debug" message it will get very very noisy, because it also prints every raw data packing coming in from the socket. 

Here I move this noisy raw data print one level down into a "trace" call - because most people will want to look at the "debug" message from hyper-h2 (what it has parsed and what state transitions, etc.). If anyone is really interested in raw on-the-wire bytestreams, then "trace" is the option to go.